codegen/Templates/python/api.mustache

```diff
@@ -27,28 +27,84 @@
 from __future__ import absolute_import
 
 import six
-from {{packageName}}.rest import ApiException
+
 from {{packageName}}.api_client import ApiClient
+from {{packageName}}.configuration import Configuration
+from {{packageName}}.rest import ApiException
 
 
 {{#operations}}
 class {{classname}}(object):
     """
     Aspose.Imaging Cloud API
 
-    :param app_key: The app key.
-    :param app_sid: The app sid.
-    :param base_url: The base URL.
-    :param api_version: API version.
     """
-    def __init__(self, app_key, app_sid, base_url=None, api_version=None):
-        self.api_client = ApiClient()
-        self.api_client.configuration.api_key['api_key'] = app_key
-        self.api_client.configuration.api_key['app_sid'] = app_sid
-        if base_url:
-            self.api_client.configuration.host = base_url
-        if api_version:
-            self.api_client.configuration.api_version = api_version
+
+    def __init__(self, app_key=None, app_sid=None, base_url=None,
```

Comment:
@s-zubov Please, go on with a review remarks of Python SDK regarding constructor definition. Thanks.